pages/common/ia.md

```diff
@@ -0,0 +1,24 @@
+# ia
+
+> Command-line tool to interact with `archive.org`.
```

Comment:
Rolls off of the tongue wrong without it imo

```suggestion
>  A command-line tool to interact with `archive.org`.
```

Reply:
I think they're both equally readable.

Reply:
They seem pretty much the same to me as well, and there are other examples of not starting the description with an article, for example ```tldr git``` returns "Distributed version control system."